manifests/base/deployments/argocd-redis_patch.yaml

```diff
@@ -21,7 +23,7 @@ spec:
         - ""
         - --appendonly
         - "no"
-        image: redis:5.0.8
```

Comment:
Any reason why we're not specifying the version?

Reply:
all versions are specified in the imagestreams